ova/provision.sh

```diff
@@ -1,56 +1,129 @@
+echo "PermitRootLogin no" >> /etc/ssh/sshd_config
+
+# Download unattended installer
+curl -so ${INSTALLER} https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCHDOC}/resources/open-distro/unattended-installation/${INSTALLER} 
+
+# Get currents version values of installer
+CURRENT_W=$(less ${INSTALLER} | grep "WAZUH_VER=")
+CURRENT_O=$(less ${INSTALLER} | grep "OD_VER=")
+CURRENT_E=$(less ${INSTALLER} | grep "ELK_VER=")
+
+# Change wazuh and documentation repository branch
+sed -i "s/uh\/[0-9]\+\.[0-9]\+\/ex/uh\/${BRANCH}\/ex/g" ${INSTALLER}
+sed -i "s/on\/[0-9]\+\.[0-9]\+\/re/on\/${BRANCHDOC}\/re/g" ${INSTALLER}
+
+# Change versions
+sed -i "s/${CURRENT_W}/WAZUH_VER=\"${WAZUH_VERSION}\"/g" ${INSTALLER}
+sed -i "s/${CURRENT_O}/OD_VER=\"${OPENDISTRO_VERSION}\"/g" ${INSTALLER}
+sed -i "s/${CURRENT_E}/ELK_VER=\"${ELK_VERSION}\"/g" ${INSTALLER}
+
+# Change repository if dev is specified
+if [ "${PACKAGES_REPOSITORY}" = "dev" ]; then
+  sed -i "s/\[wazuh\]/\[wazuh_pre_release\]/g" ${INSTALLER}
+  sed -i "s/ngpgkey\=https\:\/\/packages\.wazuh\.com/ngpgkey\=https\:\/\/packages\-dev\.wazuh\.com/g" ${INSTALLER}
+  sed -i "s/baseurl\=https\:\/\/packages\.wazuh\.com\/4\.x/baseurl\=https\:\/\/packages\-dev\.wazuh\.com\/pre\-release/g" ${INSTALLER}
+  sed -i "s/https\:\/\/packages\.wazuh\.com\/4\.x\/ui\/kibana/https\:\/\/packages\-dev\.wazuh\.com\/pre\-release\/ui\/kibana/g" ${INSTALLER}
+  sed -i "s/wazuh_kibana-[0-9\.]\+_[0-9\.]\+/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}/g" ${INSTALLER}
+fi
+
+# Execute unattended installer
+sh ${INSTALLER}
+
+# Remove installer
+rm ${INSTALLER}
+
+# Check kibana status
+until [[ "$(curl -u admin:admin -XGET https://localhost/status -I -s -k | grep HTTP)" == *"200"* ]]; do
+    echo "Waiting for Kibana..."
+    sleep 2
+done
+
+# Stop services and enable manager
+systemctl stop kibana filebeat elasticsearch wazuh-manager
+systemctl enable wazuh-manager
 
-# Setting wazuh default root password
+# Disable all host modules
+sed -i "s/<disabled>no/<disabled>yes/g" /var/ossec/etc/ossec.conf
+sed -i "s/<enabled>yes/<enabled>no/g" /var/ossec/etc/ossec.conf
 
-cp ${libraries_files}/automatic_set_ram.sh ${automatic_set_ram_location}
-chmod +x "${automatic_set_ram_location}/automatic_set_ram.sh"
-echo "@reboot . /etc/automatic_set_ram.sh" >> ram_cron
+# Disable /var/log/ files and commands monitoring
+sed -i "s/<localfile>/<!--localfile>/g" /var/ossec/etc/ossec.conf
+sed -i "s/<\/localfile>/<\/localfile-->/g" /var/ossec/etc/ossec.conf
 
+# Enable ossec_auth modules
+sed -i '/<auth>/ {N; s/<auth>.*yes/<auth>\n\ \ \ \ <disabled>no/g}' /var/ossec/etc/ossec.conf
 
-crontab ram_cron
-rm -rf ram_cron
+# Custom Login Page
+# Edit window title
+sed -i "s/null, \"Elastic\"/null, \"Wazuh\"/g" /usr/share/kibana/src/core/server/rendering/views/template.js
 
-yes wazuh | passwd root
-hostname wazuhmanager
+# Download custom files (background, logo and template)
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/6fb50e3584df557334da525324a73a5c32ae0551